src/main/java/com/healthapp/project/Service/UserService.java

```diff
@@ -31,6 +31,18 @@ public boolean isDBConnected() {
       return false;
     }
   }
+  //can probably add a method here that adds a row to make sure we're good
+    public boolean canIAddARow(){
+       try{
+           String statement = "INSERT INTO user_dex (first_name, last_name) VALUES (?,?)";
+           jdbcTemplate.update(statement, "Bo", "Katan");
+           return true;
+       } catch (Exception e){
+           System.out.println(e.getMessage());
+           e.printStackTrace();
+           return false;
+       }
+    }
```

Comment:
a quick method to see if i can write to the table also. user_dex is just a table i have in the remote db. Nothing crazy